ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml

```diff
@@ -390,6 +390,74 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-assisted-operator,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
wdyt, maybe we want to make it optional and not running by default?

Reply:
Just to put in what we talked about I'll be updating it to marking this as optional but keep the `always_run`. This will make it run but remove the required tag in github.